src/robots/mecanum.cc

```diff
@@ -13,29 +13,26 @@
 
 namespace BoBRobotics {
 namespace Robots {
-Mecanum::Mecanum(const char *path)
-  : m_Serial(path)
-  , m_Forward(0.0f)
-  , m_Sideways(0.0f)
-  , m_Turn(0.0f)
+Mecanum::Mecanum(const char *path, bool alternativeWiring)
+  : m_Serial(path), m_Forward(0.0f), m_Sideways(0.0f), m_Turn(0.0f), m_AlternativeWiring(alternativeWiring)
```

Comment:
I'd probably make it so that ``alternativeWiring`` is by default set depending on the value of an env var (either at compile time or runtime), just so we don't end up with Sussex/Sheffield specific code.